app/Actions/User/Notify.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Actions\User;
+
+use App\Facades\AccessControl;
+use App\Models\Album;
+use App\Models\Configs;
+use App\Models\Photo;
+use App\Models\User;
+use App\Notifications\PhotoAdded;
+use Illuminate\Support\Facades\Notification;
+
+class Notify
+{
+	public function do(Photo $request, $album_id = null)
+	{
+		$settings = Configs::get();
+
+		if ($settings['new_photos_notification']) {
```

Comment:
This and the two lines above need to be changed for:
```suggestion
		if (Configs::get_Value('new_photos_notification','1') == '1') {
```
This ensure that you don't re-query the config and this provides a default value.